src/github.com/getlantern/lantern-mobile/app/build.gradle

```diff
@@ -118,4 +119,14 @@ dependencies {
     compile('com.crashlytics.sdk.android:crashlytics:2.5.3@aar') {
         transitive = true
     }
+
+    apt "org.androidannotations:androidannotations:" + androidAnnotationsVersion
+    compile("org.androidannotations:androidannotations-api:" + androidAnnotationsAPIVersion )
+}
+
+apt {
+    arguments {
+        androidManifestFile variant.outputs[0].processResources.manifestFile
+        resourcePackageName 'org.getlantern.lantern'
```

Comment:
For some reason I had to change this to

```
        resourcePackageName ""
```

to get it to run on my device from Android Studio @atavism